nemo_text_processing/text_normalization/ja/data/date/month.tsv

```diff
@@ -0,0 +1,21 @@
+1	一
+01	一
```

Comment:
Since it's always the last digit that triggers the mapping, doing something like: `pynini.accep("0").ques + month_mappings` may help reduce the size of the grammar.

Reply:
Do you mean to do the mapping with pynini.delete("0") + data_file_imported_for mappings?